google-cloud-spanner/src/main/java/com/google/cloud/spanner/Type.java

```diff
@@ -127,6 +127,24 @@ public static Type pgJsonb() {
     return TYPE_PG_JSONB;
   }
 
+  /**
+   * To get the descriptor for the {@code PROTO} type.
+   *
+   * @param protoTypeFqn Proto full name
```

Comment:
Could we add an example for the type of string that we expect here?

Reply:
Added